src/main/java/org/apache/commons/io/HexDump.java

```diff
@@ -53,7 +56,26 @@ public class HexDump {
             };
 
     /**
-     * Dumps an array of bytes to an OutputStream. The output is formatted
+     * Dumps an array of bytes to an Appendable. The output is formatted
+     * for human inspection, with a hexadecimal offset followed by the
+     * hexadecimal values of the next 16 bytes of data and the printable ASCII
+     * characters (if any) that those bytes represent printed per each line
+     * of output.
+     *
+     * @param data  the byte array to be dumped
+     * @param out  the Appendable to which the data is to be written
+     *
+     * @throws IOException is thrown if anything goes wrong writing
+     *         the data to appendable
+     */
+
```

Comment:
Remove whitespace between Javadoc and code.

Reply:
Add Javadoc since tag to new public and protected elements.

Reply:
@garydgregory Thanks for reviewing. I've fixed both.